src/fsc/fscProject/fsc.fsproj

```diff
@@ -34,10 +34,6 @@
     <IntermediateOutputPath>$(ArtifactsDir)obj/$(MSBuildProjectName)/$(Configuration)/</IntermediateOutputPath>
   </PropertyGroup>
 
-  <Target Name="ValidateMibcFile" AfterTargets="Restore" BeforeTargets="Build;Pack">
-    <Error Condition="'$(Configuration)' == 'Proto' AND !Exists('$(MibcFile)')" Text="Mibc file '$(MibcFile)' does not exist." />
-  </Target>
-
```

Comment:
Why? We wan't to be sure files are there, right?

Reply:
No I am just trying out random stuff to understand why things are locked, bringing this back